index.html

```diff
@@ -8,6 +8,96 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+   <div id="home-page"> 
+        <div id="top-info">
+            <div id="top-left-info">
+                <div class="left-info">Travel</div>
+                <div class="left-info">Goo</div>
+            </div>
+            <div id="top-mid-info">
+               <div class="mid-info">Home</div>
+               <div class="mid-info">About Us</div>
+               <div class="mid-info">Packages</div>
+            </div>
+            <div id="top-right-info">
+                <img class="right-info" src="./images/star.svg">
+                <img id="circle" src="./images/profile.svg">
+            </div> 
+        </div> 
+        <div id="centralised-info">
+            <div id="mid-quote">
+                <div class="quote">Your Imagination Is</div>
```

Comment:
no need to use two divs. You can put the text inside one div and then set a maximum width for that div so that the text wraps to another line.

Reply:
but the next line should come in middle... for that reason only i do so ... if i put the text inside one div and then set a maximum width for that div the text would wrap to another line but i think it would start at same place below the above line... is what i think right ... if no let me be notified i may change...